README.md

```diff
@@ -0,0 +1,54 @@
+### Before instalating you need to have installed some programs:
```

Comment:
Since installation steps are already install.sh, I suggest to remove it from README